packages/core-data/src/hooks/constants.js

```diff
@@ -0,0 +1,4 @@
+export const IDLE = 'IDLE';
+export const RESOLVING = 'RESOLVING';
+export const ERROR = 'ERROR';
+export const SUCCESS = 'SUCCESS';
```

Comment:
I think we can rewrite this file into typescript either using `enums` or `const`:


```ts
export enum Status {
  Idle = 'IDLE',
  Resolving = 'RESOLVING',
  Error = 'ERROR',
  Success = 'SUCCESS',
};
```